src/main/java/dev/codesoapbox/dummy4j/dummies/identifier/internationalnumber/checkdigitformulas/CheckDigitFormulaProvider.java

```diff
@@ -0,0 +1,27 @@
+package dev.codesoapbox.dummy4j.dummies.identifier.internationalnumber.checkdigitformulas;
+
+import dev.codesoapbox.dummy4j.dummies.shared.checkdigitformulas.LuhnFormula;
+
+public class CheckDigitFormulaProvider {
```

Comment:
Since we're only using it in this package, it should be called something like `InternationalNumberCheckDigitFormulaProvider`.